### ISSUE TITLE
Can I train my own by only using RGB features?

### ISSUE THREAD

**Issue by yuanqzhang** [frostinassiky/gtad#21]
I have my own dataset pre-trained by I3D model on Kinetics, and output the result of avg-pool layer with 1024 dims. The length of the list is processed after frame extraction.
Can I only use this RGB features as my input of the model, as the extraction of optical flow will cost too much time?
 Many thanks for the time.

**Comment by frostinassiky** [comment]
Hi @yuanqzhang . Thanks for your interest in G-TAD. 

I agree with you that extracting the optical flow takes too much time. You can do as you described: only take the RGB feature as model input.

**Comment by ZhengLeon** [comment]
Hi @frostinassiky 
I wonder how to get the uNet_test.npy for my own dataset?
Is it come from the output proposals by your model and  the UntrimmedNet?

Thanks a lot!

**Comment by frostinassiky** [comment]
Dear @ZhengLeon 
`uNet_test.npy` contains the classification results for each video in the test set. You can use any video classification model to produce the prediction. See #10

**Comment by ZhengLeon** [comment]
Thanks so much for your response! @frostinassiky

**Comment by yuanqzhang** [comment]
many thanks

**Comment by Shuhan136** [comment]
> I have my own dataset pre-trained by I3D model on Kinetics, and output the result of avg-pool layer with 1024 dims. The length of the list is processed after frame extraction.
> Can I only use this RGB features as my input of the model, as the extraction of optical flow will cost too much time?
> Many thanks for the time.


can you tell me how can i get the result of avg-pool layer with 1024 dims from RGB frame or Optical flow frame?
